arviz/tests/test_data.py

```diff
@@ -270,6 +270,32 @@ def get_inference_data(self):
                                 }
                            )
 
+    def get_inference_data2(self):
```

Comment:
didn't we already merge this?

Reply:
Yeah, artifact from rebasing. No idea why it stayed, but didn't have time to fix it (I didn't want to break this pr).